ddw_analyst_ui/settings.py

```diff
@@ -99,7 +99,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'postgres',
+        'NAME': 'analyst_ui',
```

Comment:
There's two `settings.py` files. This is the main one you can use locally, then there's one for `Docker` (. Please make this update to the `Docker` file as well `docker_settings.py`.

Reply:
Second settings was added only for migration/inspectdb activities which wasn't a managed process (docker's scope), wont be necessary though since I'm moving the models to be managed by django